### ISSUE TITLE
Crash on start up

### ISSUE THREAD

**Issue by TheFelixLive** [wiiu-smarthome/homebridge-wiiu#2]
**Describe The Bug:**
When I start/restart the Homebridge server, the Wii U doesn't  appear in Homebridge and in the Home app. Instadt Homebridge gives me the following errors, listed in Related Logs

**To Reproduce:**
I want to try the Plugin and gave me this error, so try to install with the associated hardware mentioned in Environment

**Expected behavior:**
It simply should run.

**Related Logs:**
```
[16/09/2024, 19:19:31] ====================
[16/09/2024, 19:19:31] ERROR LOADING PLUGIN homebridge-wiiu:
[16/09/2024, 19:19:31] Error: Invalid package config /var/lib/homebridge/node_modules/homebridge-wiiu/node_modules/axios/package.json while importing "axios" from /var/lib/homebridge/node_modules/homebridge-wiiu/dist/platform.js. Unterminated string in JSON at position 7671
    at Object.read (node:internal/modules/package_json_reader:83:15)
    at packageResolve (node:internal/modules/esm/resolve:835:45)
    at moduleResolve (node:internal/modules/esm/resolve:927:18)
    at defaultResolve (node:internal/modules/esm/resolve:1169:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:540:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:509:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:96:40)
    at link (node:internal/modules/esm/module_job:95:36)
[16/09/2024, 19:19:31] ====================
[16/09/2024, 19:19:31] Loading 3 platforms...
[16/09/2024, 19:19:31] No plugin was found for the platform "HomebridgeWiiUPlugin" in your config.json. Please make sure the corresponding plugin is installed correctly.
```

**JSON config file:**
```json
{
    "ip": "Wii U IP:8572",
    "name": "Wii U",
    "platform": "HomebridgeWiiUPlugin"
}
```
> Wii U IP is replaced with an actual IP-Address

**Environment:**

* **Plugin Version**: 1.0.0
* **Homebridge Version**:  v1.8.4 (HAP v0.12.2)
* **Node.js Version**: 20.17.0
* **NPM Version**: Is this relevant to the error and if so, where can I find it?
* **Device**: Raspberry Pi Zero 2 W
* **Operating System**: Raspbian 64 Lite

**Comment by JPeisach** [comment]
JSON needs to be formatted in a way that has proper indentation. For example, this is a part of my testing config.json:
```json
        {
            "ip": "192.168.1.195:8572",
            "platform": "HomebridgeWiiUPlugin"
        },
 ```

Notice how there are indentations. Your config file needs that too.

**Comment by TheFelixLive** [comment]
My file actually had the indentations in it but I messed up something when formatting (that way the log is from 19 o'clock).
I also tried **yours**

```json
        {
            "ip": "Wii U IP:8572",
           "platform": "HomebridgeWiiUPlugin"
        },
```

> Wii U IP is replaced with an actual IP-Address

and I got the same error.

**Comment by JPeisach** [comment]
Just share your config file (omit or put in foo for parts that are actually needing to be kept secret like API keys). The Wii U IP address is local, only people on your network can access it. There's no harm in sharing it.

Here is the config.json of the homebridge instance I use for testing:

```json
{
    "bridge": {
        "name": "Homebridge ABC3",
        "username": "0E:49:81:3A:AB:CB",
        "port": 51070,
        "pin": "318-08-913",
        "advertiser": "bonjour-hap"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "ip": "192.168.1.195:8572",
            "platform": "HomebridgeWiiUPlugin"
        },
        {
            "name": "Google Smart Home",
            "token": "(token)",
            "notice": "Keep your token a secret!",
            "platform": "google-smarthome"
        }
    ],
    "disabledPlugins": [
        "homebridge-gsh"
    ]
}
```

**Comment by TheFelixLive** [comment]
Sorry for the long waiting but finaly found the issue. After some updates I got this error:
>     [11/12/2024, 21:35:13] Error: ENOENT: no such file or directory, open './titles.json'
>      at Object.readFileSync (node:fs:441:20)
>      at new WiiUPlatformAccessory (file:///var/lib/homebridge/node_modules/homebridge-wiiu/src/platformAccessory.ts:95:21)
>      at WiiUPlatform.discoverDevices (file:///var/lib/homebridge/node_modules/homebridge-wiiu/src/platform.ts:121:7)
>      at HomebridgeAPI.<anonymous> (file:///var/lib/homebridge/node_modules/homebridge-wiiu/src/platform.ts:46:12)
>      at HomebridgeAPI.emit (node:events:536:35)
>      at HomebridgeAPI.signalFinished (/var/lib/homebridge/node_modules/homebridge/src/api.ts:275:10)
>      at Server.start (/var/lib/homebridge/node_modules/homebridge/src/server.ts:178:14)
So it says somethink about missing tiles (I think [theses](http://wiiubrew.org/wiki/Title_database) one) and I have some wii u virtual console injection titles (all on a USB drive). I unplugged it and guess what, it worked (untill I replugg the USB drive in to the console). Strangely enough, the console no longer appeared in Homekit since than.
My current status after resetting homebrighe is again the titles error. Is there a way to read those titles via ftpiiu's internal file system and create an owne titles.json file?

**Comment by JPeisach** [comment]
Shoot. I forgot to make the titles.json file if it doesnt exist.

The titles.json is not on your Wii U, it's on your server running Homebridge :)

**Comment by JPeisach** [comment]
https://github.com/ItzSwirlz/homebridge-wiiu/commit/191324355b5e2abe700632639679777089d5a8cf

Actually I did sneak in the fix, I just never released it. Whoops. I'll send it out shortly.

**Comment by JPeisach** [comment]
Fixed in v1.1.1 - sorry for the wait
